llm_memory/components/memory_sql_manager.py

```
@@ -1795,6 +1795,28 @@ def _get_memories_by_ids_sync(self, memory_ids: List[str]) -> List[BaseMemory]:
+        try:
+            with self._db_lock:
+                conn = self._get_conn()
+                rows = conn.execute(
+                    "SELECT mr.id FROM memory_records mr "
+                    "JOIN memory_tag_rel mtr ON mtr.memory_id = mr.id "
+                    "JOIN global_tags gt ON gt.id = mtr.tag_id "
+                    "WHERE gt.name = ? ORDER BY mr.created_at DESC LIMIT 200",
+                    (tag,),
+                ).fetchall()
+            return [str(row[0]) for row in rows]
+        except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)